src/win32/pthread/pthread_mutex_init.c

```diff
@@ -39,7 +39,9 @@
 #include <Objbase.h>
 #include <assert.h>
 
+#if REALM_UWP
 int _getpid(void);
+#endif
```

Comment:
_getpid() is new and was introduced by another UWP PR. So this #if actually restores original behaviour